### ISSUE TITLE
JIRA comments being posted for unrelated emails

### ISSUE THREAD

**Issue by stevegoossens** [stevegoossens/CommsTool2#1]
The wrong Outlook send event can be caught, and results in a JIRA post being made with a screenshot of an non-comms email.

There needs to be some control/determination to filter these Outlook send events.

**Comment by stevegoossens** [comment]
The simplest solution was to evaluate the recipients (To) field of a send event.  If it is the same as the sender we use in the app (digital.247operations@bbc.co.uk), then we can safely assume it is a comms email, as there is no other scenario (that I can think of) where we would send an email from the group inbox to ourselves (as the sole recipient).  This is the determining logic for whether or not the sent email is a comms email, it only proceeds to screenshot/comment if it is deemed a comms email.

Two things to note:

1. The resolved GAL recipient for digital.247operations@bbc.co.uk (appearing as Digital 24/7 Operations in the email's To field) does not return an email address via Microsoft.Office.Interop.Outlook.Recipient.Address.  Instead it returns "/O=BBC/OU=MAIN/cn=Recipients/cn=FMTOps".  I had to instead get the email address via Microsoft.Office.Interop.Outlook.Recipient.PropertyAccessor (using a tag for SMTP address).
2. The SMTP address returned was "Digital.247Operations@bbc.co.uk", so I converted that and the app's sender variable to lowercase for the equality test.
